itou/templates/apply/includes/buttons/new_diagnosis.html

```diff
@@ -1,5 +1,13 @@
 <p>
-    <a href="{% url 'apply:eligibility' job_application_id=job_application %}" class="btn btn-primary btn-block btn-ico">
-        <span>Valider les critères d'éligibilité</span>
-    </a>
+    {% with suspension=job_application.to_siae.get_active_suspension_text_with_dates %}
+        {% if request.user.is_siae_staff and suspension %}
+            <span data-toggle="tooltip" data-placement="top" title="Vous ne pouvez pas valider les critères d'éligibilité suite aux mesures du contrôle a posteriori. {{ suspension }}">
+                <button class="btn btn-primary btn-block btn-ico disabled">Valider les critères d'éligibilité</button>
```

Comment:
Il faut tout de même indiquer au navigateur que le bouton est désactivé. Autrement, on peut le focus (avec Tab).
```suggestion
                <button class="btn btn-primary btn-block btn-ico disabled" disabled>Valider les critères d'éligibilité</button>
```

Reply:
Je suis partagé car si on ne peut pas focus le bouton, on ne peut pas faire apparaître le tooltip (sans souris ou touch)...

Reply:
C’est pourquoi je pense que l’icône interactive à côté est une meilleure idée que le tooltip sur l’élément disabled ;)